queries/rust/highlights.scm

```diff
@@ -84,7 +87,7 @@
 (metavariable) @function.macro
 
 ;; Attribute macros
-(meta_item (identifier) @function.macro)
+(attribute_item (meta_item (identifier) @function.macro))
```

Comment:
otherwise this was matching all identifiers inside the macro

![Screenshot from 2021-09-30 22-17-18](https://user-images.githubusercontent.com/4975310/135560083-9e508a81-e25c-4c27-a7cf-8f48a9de08ab.png)